src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const personPartnerType = person.sex === 'm' ? 'wife' : 'husband';
```

Comment:
it's a good practice to use object destructing, so you won't need to use `person.`
```suggestion
export const Person = ({ person }) => {
  const {...} = person
  const personPartnerType = person.sex === 'm' ? 'wife' : 'husband';
```